### PR TITLE
InfluxDB Integration

### PR DESCRIPTION
**Pull request by StanGirard**
# InfluxDB

## Introduction

We all have been seeking  better analytics for Instapy.
I decided to Integrate InfluxDB which is a No-SQL time series database.
Inputs are based on the time.

## Implementation

I created a Singleton Class in settings which creates a connection to the db.
It is initialised in instapy.py
I also created functions that you can use to add values in the db or change db.
Functions can be added very easily

## How to use

To use the SingletonClass just import `from .settings import InfluxDBLog`
Then in your file just call ÌnfluxDBLog().addEntry(args)` to add a new entry in the DB.

## Prerequesites

in order to have this working please install InfluxDB on your computer and create a user.
For it to be activated you must in the instapy constructor input your InfluxDB username, password, host, port and database name.

## Visualize Data

In order to visualise the data just install Grafana and create dashboards.
It could end up looking like this 

capture d ecran 2019-02-23 a 00 12 03

### CODE REVIEW

**Comment by StanGirard** [comment]
@breuerfelix  Merge issues fixed

File: instapy/instapy.py [hunk 1]
```diff
@@ -100,6 +101,12 @@ def __init__(self,
                  disable_image_load=False,
                  bypass_suspicious_attempt=False,
                  bypass_with_mobile=False,
+                 user_influx=None,
+                 password_influx=None,
+                 db_influx=None,
+                 host_influx=None,
+                 port_influx=None,
+                 multi_logs=True,
```
Comment (timgrossmann) [inline instapy/instapy.py:109]: Duplicated `multi_logs` here, sorry 😅

Comment (breuerfelix) [reply instapy/instapy.py:109]: is this fixed ?  
/edit

just realised its not fixed, travis showing errors :D  
@StanGirard could you fix that please ?:))

Comment (StanGirard) [reply instapy/instapy.py:109]: @breuerfelix  Done 👍

**Pull request merged**